python/main.py

```diff
@@ -41,5 +74,52 @@ async def get_image(image_name):
     if not image.exists():
         logger.debug(f"Image not found: {image}")
         image = images / "default.jpg"
-
     return FileResponse(image)
+
+# JSONファイルから商品データを読み込む関数
+def load_items() -> List[Dict]:
+    with open("items.json", "r") as file:
+        data = json.load(file)
+    return data["items"]
+
+@app.get("/items/{item_id}")
+def read_item(item_id: str):
+    items = load_items()
+    for item in items:
+        if item["id"] == item_id:
+            return item
+    raise HTTPException(status_code=404, detail="Item not found")
+
+
+@app.get("/")
+def read_root():
+    return {"message": "Hello, world!"}
+
+import sqlite3
+
+# JSONファイルからデータを読み込む
+with open('items.json', 'r') as f:
+    items_data = json.load(f)
+
+# SQLite3データベースに接続する
+conn = sqlite3.connect('db/mercari.sqlite3')
+cursor = conn.cursor()
+
+# itemsテーブルを作成する
+cursor.execute('''CREATE TABLE IF NOT EXISTS items (
+                    id INTEGER PRIMARY KEY,
+                    name TEXT,
+                    category TEXT,
+                    image_name TEXT
+                )''')
+
+# 読み込んだデータをデータベースに挿入する
+for item in items_data:
+    cursor.execute('''INSERT INTO items (name, category, image_name)
+VALUES (?, ?, ?)''', (item['name'], item['category'], item['image_name']))
+
+# 変更をコミットする
+conn.commit()
+
+# 接続を閉じる
+conn.close()
```

Comment:
(no need to delete but just fyi) these changes are for step4, right? It is better to contain related changes only.